extension/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
hmm, why run it through the es2015 preset? Except for maybe the import this all looks like it'll run in stable chrome